1. Desktop app calls Web API/TodoListService/appsettings.json

```diff
@@ -10,13 +10,12 @@
     "Domain": "qualified.domain.name", // for instance contoso.onmicrosoft.com. Not used in the ASP.NET core template
     "TenantId": "common" // The tenant's Directory ID (a guid) or 'common' or 'organizations' or 'consumers'
   },
-  "Kestrel": {
-    "Endpoints": {
-      "Http": {
-        "Url": "http://localhost:44351"
-      }
```

Comment:
This is the setting that was causing the breaks when using .NET 6.0

At first I thought it was unsupported but now I'm seeing some conflicting documentation.